src/modules/navigation/MainNavigator.js

```diff
@@ -33,108 +13,26 @@ const MainNavigator = () => {
-            fontFamily: Typography.FONT_FAMILY_BOLD,
-            paddingBottom: 7,
-          },
-          headerRight: () => (
-            <FontForgeIcon
-              name="doctor_profile"
-              size={30}
-              color={Colors.PURPLE_VERY_LIGHT}
-              style={{
-                marginRight: 20,
-                marginBottom: 7,
-              }}
-            />
-          ),
-          headerBackImage: () => {
-            return (
-              <TouchableOpacity
-                onPress={() =>
-                  Alert.alert(
-                    "",
-                    "Czy na pewno chcesz przerwać wywiad i powrócić do listy pacjentów?",
-                    [
-                      {
-                        text: "Kontynuuj",
-                        onPress: () => {},
-                        style: "cancel",
-                      },
-                      {
-                        text: "Przerwij",
-                        onPress: () => navigation.navigate("PatientsList"),
-                      },
-                    ]
-                  )
-                }
-              >
-                {backIcon}
-              </TouchableOpacity>
-            );
-          },
-        })}
+        screenOptions={{ ...HeaderOptions }}
       >
         <Stack.Screen
           name="PatientsList"
-          component={PatientsListScreen}
+          component={PatientsList}
           options={{ title: "Lista pacjentów" }}
         />
         <Stack.Screen
           name="PatientCard"
-          component={PatientCardScreen}
-          options={({ navigation }) => ({
-            title: "Karta pacjenta",
-            headerBackImage: () => {
-              return (
-                <TouchableOpacity
-                  onPress={() => navigation.navigate("PatientsList")}
-                >
-                  {backIcon}
-                </TouchableOpacity>
-              );
-            },
-          })}
-        />
-        <Stack.Screen
-          name="AddPatient"
-          component={AddPatientScreen}
-          options={{
-            title: "Dane osobowe",
-          }}
-        />
-        <Stack.Screen
-          name="BasicData"
-          component={BasicDataScreen}
+          component={PatientCard}
           options={{
-            title: "Informacje podstawowe",
-          }}
-        />
-        <Stack.Screen
-          name="PhysicalExamination"
-          component={PhysicalExaminationScreen}
-          options={{
-            title: "Badanie fizykalne",
+            title: "Karta pacjenta",
           }}
         />
         <Stack.Screen
-          name="PsychiatricAssessment"
-          component={PsychiatricAssessmentScreen}
-          options={{
-            title: "Badanie psychiatryczne",
-          }}
+          name="Registration"
+          component={Navigator}
+          options={({ route }) => ({
+            headerTitle: getHeaderTitle(route),
```

Comment:
coś to nie działa, na każdym ekranie z rejestracji pacjenta wyświetla się "Dane osobowe"